js/slider.js

```diff
@@ -0,0 +1,30 @@
+// setting up an index
```

Comment:
This javascript makes the imageslider work, as I commented before it was done now because of extra time